maat-court-data-api/src/test/java/gov/uk/courtdata/dces/mapper/ContributionFileMapperTest.java

```diff
@@ -23,38 +26,76 @@ class ContributionFileMapperTest {
     void fdcMappingTest(){
         CreateFdcFileRequest request = createFdcRequest();
         ContributionFilesEntity mapped = mapper.toContributionFileEntity(request);
-        assertTrue(validateMappedObject(mapped, request),"Mapping the fdc request was incorrect.");
+        assertValidMappedObject(mapped, request);
     }
 
     @Test
     void contributionMappingTest(){
         CreateContributionFileRequest request = createContributionRequest();
         ContributionFilesEntity mapped = mapper.toContributionFileEntity(request);
-        assertTrue(validateMappedObject(mapped, request),"Mapping the fdc request was incorrect.");
+        assertValidMappedObject(mapped, request);
 
     }
 
-    private boolean validateMappedObject(ContributionFilesEntity mapped, CreateFileRequest request){
+    @Test
+    void testEntityToContributionFileResponse() {
+        var entity = TestEntityDataBuilder.getPopulatedContributionFilesEntity(99);
+        var mapped = mapper.toContributionFileResponse(entity);
+        assertValidMappedObject(mapped, entity);
+    }
+
+    @Test
+    void testEntityToContributionFileErrorResponse() {
+        var entity = TestEntityDataBuilder.getContributionFileErrorsEntity(99, 888);
+        var mapped = mapper.toContributionFileErrorResponse(entity);
+        assertValidMappedObject(mapped, entity);
+    }
+
+    private static void assertValidMappedObject(ContributionFilesEntity mapped, CreateFileRequest request){
         assertEquals(request.getXmlContent(), mapped.getXmlContent());
```

Comment:
When making a large number of assertions it would make it easier for future developers to use the JUnit assertAll so all test failures are shown instead of only the first failure.

e.g.
```
  private static void assertValidMappedObject(ContributionFilesEntity mapped, CreateFileRequest request) {
    assertAll(() -> assertEquals(request.getXmlContent(), mapped.getXmlContent()),
        () -> assertEquals(request.getAckXmlContent(), mapped.getAckXmlContent()),
        () -> assertEquals(request.getRecordsSent(), mapped.getRecordsSent()),
        () -> assertEquals(request.getXmlFileName(), mapped.getFileName()),
        // validate non-map-related fields are unset. No stray mappings.
        () -> assertNonMappedAreNull(mapped.getFileId()),
        () -> assertNonMappedAreNull(mapped.getRecordsReceived()),
        () -> assertNonMappedAreNull(mapped.getDateCreated()),
        () -> assertNonMappedAreNull(mapped.getDateModified()),
        () -> assertNonMappedAreNull(mapped.getUserModified()),
        () -> assertNonMappedAreNull(mapped.getDateSent()),
        () -> assertNonMappedAreNull(mapped.getDateReceived()),
        // should have a default of DCES for the created.
        () -> assertEquals("DCES", mapped.getUserCreated()));
  }
```

Reply:
makes sense. done